File: Odd or Even Number/OoE.c
#include<stdio.h>
void main()
{
	int a;
	printf("Enter a number : ");
	scanf("%d",&a);
	if(a%2!=0)
	{
		printf("%d is an Odd Number",a);
	}
	else
	{
		printf("%d is an Even Number",a);
	}
}

File: Problem Statements/Odd or Even Number/OoE.java
import java.util.Scanner;
class OoE{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a number : ");
        int a=sc.nextInt();
        sc.close();
        if(a%2!=0){
            System.out.println(a+" is a Odd Number");
        }else{
            System.out.println(a+" is a Even Number");
        }
    }
}